src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -64,4 +71,17 @@ public void initialize(URL url, ResourceBundle resourceBundle) {
             System.err.println("User wasn't started");
         }
     }
+
+    @FXML
+    void setUserName() {
+        userNameText = userName.getText();
+    }
+
+    public String combineUserNameWithMessage(String message, String userName){
+        if (userName.isEmpty() || userName == null){
+            return "[]- " + message;
+        }else {
+            return String.format("[%s]- %s", userName, message);
+        }
+    }
```

Comment:
Change this to add it as a signature. Something like
_Text_ | _Signature_
If signature is emptry add nothing

Reply:
`public String combineUserNameWithMessage(String message, String signature){
        if (signature.isEmpty()){
            return message;
        }
        return String.format("%s | %s", message, signature);
    }`
this would be my solution